### PR TITLE
DOCSP-1003/1103: Change 'Using MongoDB' Section Layout & Fix Vertical Alignment

### PR DESCRIPTION
**Pull request by nlarew**
jira: https://jira.mongodb.org/browse/DOCSP-1003

stage: https://docs-mongodbcom-staging.corp.mongodb.com/landing/nick/DOCSP-1003/index.html#using-mongodb

Primary Changes:

- Section titles are aligned vertically in two-columns.
- Re-ordered several sections for better vertical alignment.
- Removed `MongoDB Manual 3.4` section because it only has one entry that is also in the cards at the top of the page.

Considerations:

- Removing `MongoDB Manual 3.4` left an odd number of sections so the bottom row now has only one section. Should we change up the sections to go back an even number?
  - A candidate for removal is `Cloud` because it also has a card at the top of the page. It also has a link to Atlas which has another link (with a different description) in the `Monitoring` section.
- Most sections have descriptive text for each heading but three do not (`Client Libraries (Drivers)`, `Manage Users`, and `Security`). Should we add descriptions to these, remove descriptions from all links, or just leave it as is?
  - `Client Libraries (Drivers)` is the longest section even without descriptions and the links are mostly self-explanatory.
  - `Manage Users` and `Security` could probably have descriptions added.

### CODE REVIEW

File: src/html/home.html [hunk 1]
```diff
@@ -66,436 +66,419 @@
     </ul>
   </div>
 
-  <div class="main__content main-column">
```
Comment (jdestefano-mongo) [inline src/html/home.html]: marian.js uses the 'main-column` and `main__cards` classes.

Please verify:
1. Removal of those are safe here.
2. Are these class names used in other pages in other docs properties or marian search result html files? If not should we update that file?

File: src/html/home.html [hunk 2]
```diff
@@ -66,436 +66,419 @@
...
-              <a class="block__link" href="https://docs.cloudmanager.mongodb.com/">MongoDB Cloud Manager</a>
-            </h4>
-            <p class="block__body">Reference guide for MongoDB Cloud Manager. Learn
-            how to manage your MongoDB deployments using Cloud Manager.</p>
-          </div>
+  <div class="main__content">
+    <h1 class="title">Welcome to the MongoDB Docs</h1>
+
+    <a class="card" href="https://docs.mongodb.com/manual/tutorial/getting-started/">
+      <span class="card__icon card__icon--document"></span>
```
Comment (jdestefano-mongo) [inline src/html/home.html]: These icons are not showing up in staging.

File: src/html/home.html [hunk 3]
```diff
@@ -66,436 +66,419 @@
...
+
+    <h2>
+      <span class="anchor" id="using-mongodb">&nbsp;</span>
+      Using MongoDB
+    </h2>
+
+    <div class="row">
+      <div class="col-3 no-left-gutter">
+        <div class="block">
+          <h3 class="block__header">Installation & Upgrading</h3>
```
Comment (jdestefano-mongo) [inline src/html/home.html]: My preference would be to keep "and" instead of the '&'

File: src/html/home.html [hunk 4]
```diff
@@ -66,436 +66,419 @@
...
+          </h4>
+          <p class="block__body">How to upgrade to the latest version of MongoDB.</p>
+
+        </div>
+      </div>
+      <div class="col-3 no-right-gutter">
+        <div class="block">
+          <h3 class="block__header">Client Libraries (Drivers)</h3>
+
+          <h4 class="block__title block__title--only">
```
Comment (jdestefano-mongo) [inline src/html/home.html]: The `block__title--only` used to give a little extra bottom margin (10px vs. 5px) on links that did not have a text paragraph following it.  It appears this only worked when the HTML was structured <a><h4>Link text</h4><a>.

I think this actually looks better WITHOUT the extra padding applied (at 5px) with the new format. If we apply this format to the "Cloud" and "Tools" landing pages we should remove this from the css.

**Comment by jdestefano-mongo** [comment]
Yeah, open/expanded by default works too if that's a quick fix.  Good call.

File: src/html/home.html (hunk 1 above)
Comment (nlarew) [reply src/html/home.html]: Oops, didn't intend to remove `main-column`. Must have accidentally slipped through in a merge.

File: src/html/home.html (hunk 2 above)
Comment (nlarew) [reply src/html/home.html]: The missing icons seem to be a quirk of our staging process. As far as I can tell the `images` folder is not actually uploading to s3 after running `make stage`, not sure why.

File: src/html/home.html (hunk 3 above)
Comment (nlarew) [reply src/html/home.html]: Reverted "&" -> "and"

File: src/html/home.html (hunk 4 above)
Comment (nlarew) [reply src/html/home.html]: Removed `block__title--only` from all three landing pages.

**Comment by nlarew** [comment]
Update per Jon's comments. For `cloud` and `tools` I tried out both one- and two-column layouts and two-column seems to make more sense despite the lower number of sections. The two-column layouts are staged.

See below for screenshots of the one-column layout on both `cloud` and `tools`. In my opinion there's too much empty space on the right side of these.

screen shot 2017-09-18 at 1 07 43 pm
screen shot 2017-09-18 at 1 25 17 pm

File: src/html/home.html (hunk 4 above)
Comment (jdestefano-mongo) [reply src/html/home.html]: Can you remove it from the sass/css too if it's no longer used anywhere?

**Comment by jdestefano-mongo** [comment]
Thanks for providing those screenshots.  I agree that it looks too empty with the single column since some of those products do not have descriptions for the links.  The only thing that isn't great is that under tools BI connector has descriptions and the others do not.  I'd like to loop @meltothewong one more time to get her input, but overall I think it's still better than what we had since users totally avoided this part of the site.  Thoughts?

Other things:
- Please remove the `block__title--only` from the sass/css if it's no longer used anywhere.
- Double check `main__cards` is not needed.  This was in `marian.js`, I saw your note but didn't see this added back in.

**Comment by meltothewong** [comment]
Thanks for looping me in @jdestefano-mongo. Agreed, it looks a bit empty. Sorry for the back and forth. Can we treat this section like the homepage and make it a two-col with the same vertical rhythm?

**Comment by jdestefano-mongo** [comment]
Staging URLs for Cloud and Tools:

https://docs-mongodbcom-staging.corp.mongodb.com/landing/nick/DOCSP-1003/cloud/

https://docs-mongodbcom-staging.corp.mongodb.com/landing/nick/DOCSP-1003/tools/

**Comment by meltothewong** [comment]
Thanks @jdestefano-mongo! Moving our thread to here.

After looking at it on staging, Jon and I noticed there's a weird vertical rhythm happening within the paragraphs (even on the homepage). Can we clean up the typography by doing the following across all pages:

`h3 {
font-weight: normal or 300;
}
`

`p or block__body {
line-height: 24px;
}
`

I also want to add another small request, while we're tightening up the layout. The `.card` class has some margin-padding issues where it doesn't align with the rest of the content. 
screen shot 2017-09-19 at 4 50 45 pm

Can we tighten that up by changing the padding to `margin: 15px 30px 15px 0;`?

Thanks!

**Comment by nlarew** [comment]
Updated per feedback:

- Added `main__cards` div back to `home.html`
- Removed`block__title--only` from css
- Removed `line-height: 22px;` from `block__body` and `card__body`
   Added `line-height: 24px;` to `p`
- Changed `font-weight` to `normal` from `bold` for `h3`
  Added `font-weight: bold;` to `block__header`
  - Adding bold to `block__header` preserves the bold block titles without bolding the card titles.

**Comment by nlarew** [comment]
Added DOCSP-1103 to this PR

**Comment by meltothewong** [comment]
This looks so clean! 😭  Great job, @nlarew!

**Comment by nlarew** [comment]
Closing this pull request in favor of https://github.com/mongodb/docs-landing/pull/15

**Pull request closed without merging**